stm/vernac_classifier.ml

```diff
@@ -66,10 +60,7 @@ let rec classify_vernac e =
     (* Nested vernac exprs *)
     | VernacProgram e -> classify_vernac e
     | VernacLocal (_,e) -> classify_vernac e
-    | VernacPolymorphic (b, e) -> 
-      if b || Flags.is_universe_polymorphism () (* Ok or not? *) then
-	make_polymorphic (classify_vernac e)
-      else classify_vernac e
+    | VernacPolymorphic (b, e) -> classify_vernac e
```

Comment:
how is this related to this PR?

Reply:
Look above, `make_polymorphic` is gone. What was that supposed to mean?